extensions/amp-pan-zoom/validator-amp-pan-zoom.protoascii

```diff
@@ -29,6 +29,10 @@ tags: {  # <amp-pan-zoom>
   html_format: EXPERIMENTAL
   tag_name: "AMP-PAN-ZOOM"
   requires_extension: "amp-pan-zoom"
+  attrs: {
+    name: "disable-double-tap"
```

Comment:
please alphabetize by attrs.name (e.g. after `controls`)

Reply:
Oh oops. Idk why I misread the alphabet here.